### PR TITLE
Move RPM sensor logging into AP_RPM

### PR DESCRIPTION
**Pull request by peterbarker**
We could potentially move the message definition back into here too.

Tested by running the Helicopter autotests and noting that `RPM` is still present in the logs:

image

### CODE REVIEW

File: Blimp/Blimp.h [hunk 1]
```diff
@@ -416,8 +416,6 @@ class Blimp : public AP_Vehicle
     void read_rangefinder(void);
     bool rangefinder_alt_ok();
     bool rangefinder_up_ok();
-    void rpm_update();
-    void update_optical_flow(void);
```
Comment (hendjoshsr71) [inline Blimp/Blimp.h:420]: Unintentional remove here? `void update_optical_flow(void);`

**Review by hendjoshsr71 — CHANGES_REQUESTED** [review]

File: Blimp/Blimp.h (hunk 1 above)
Comment (peterbarker) [reply Blimp/Blimp.h:420]: Nope, just trying to sneak it in.  You'll note the commit message is generic :-)

Comment (hendjoshsr71) [reply Blimp/Blimp.h:420]: I guess we will let you have it ;)

**Review by hendjoshsr71 — APPROVED** [review]
LGTM

**Review by IamPete1 — APPROVED** [review]

**Pull request merged**